seatunnel-server/seatunnel-spi/src/main/java/org/apache/seatunnel/spi/scheduler/dto/InstanceDto.java

```diff
@@ -26,7 +26,7 @@
 @Builder
 public class InstanceDto {
     private long instanceId;
-    private long instanceCode;
+    private long jobId;
     private String instanceName;
     private Date submitTime;
     private Date startTime;
```

Comment:
Data transmission should used the type of packaging

Reply:
this dto is just used to transform data internally and these field would nerver be null